tests/dummy/app/services/application.js

```diff
@@ -0,0 +1,65 @@
+import Ember from 'ember';
+import authConfig from '../models/authConfig';
+
+/* loaded in `index.html` */
+const platformClient = require('platformClient');
```

Comment:
Does this have to be a `require`? just curious.

Reply:
I think so. I tried to get ember to build the dep into the bundle, but couldn't (with platformClient and streaming-client). This is how the docs say to grab the platform client from the window. Don't know why, but this is the only way it works.